astropy_helpers

```diff
@@ -1 +1 @@
-Subproject commit 3e044eb26395c8569e3e15bf821c6ae5eeea324a
+Subproject commit fedd7c52ba80e2246f83b8d1752baa3a153a33e2
```

Comment:
Yes, that accidentally slipped through (it usually stays out of my way; looks like I needed to update it).